app/Providers/ViewServiceProvider.php

```diff
@@ -55,7 +55,7 @@ public function boot()
             $authUser = auth()->user();
             $view->with('authUser', $authUser);
             $view->with('spidAuthUser', app()->make('SPIDAuth')->getSPIDUser());
-            $view->with('showActivePublicAdministration', session()->has('tenant_id') && $authUser && $authUser->activePublicAdministrations->where('id', session('tenant_id')) && (1 === $authUser->activePublicAdministrations->count()));
+            $view->with('showPublicAdministration', session()->has('tenant_id') && (1 === $authUser->publicAdministrations->count()));
```

Comment:
Qui si corregge un bug per il quale solo le pa attive erano conteggiate per calcolare il flag relativo alla visualizzazione del nome della pa nell'header.

Reply:
In questo modo vengono conteggiate anche le sospese.